pkgs/servers/x11/xorg/tarballs.list

```diff
@@ -218,4 +218,4 @@ mirror://xorg/individual/util/lndir-1.0.3.tar.bz2
 mirror://xorg/individual/util/makedepend-1.0.6.tar.bz2
 mirror://xorg/individual/util/util-macros-1.19.3.tar.bz2
 mirror://xorg/individual/util/xorg-cf-files-1.0.7.tar.bz2
-mirror://xorg/individual/xserver/xorg-server-21.1.3.tar.xz
+mirror://xorg/individual/xserver/xorg-server-1.20.13.tar.xz
```

Comment:
you forgot to change this to .14, fix in https://github.com/NixOS/nixpkgs/pull/199912